btmn_106.nv.json

```diff
@@ -0,0 +1,47 @@
+{
+	"_notes": ["Compiled by Horsepin."],
+	"_copyright": "Copyright (C) 2016 by Tom Collins <tom@tomlogic.com>",
+	"_license": "GNU Lesser General Public License v3.0",
+	"_endian": "big",
+	"_roms": ["btmn_106"],
+	"_fileformat": 0.1,
+	"_version": 0.1,
+	"last_game": [
+		{"start": "1479", "encoding": "bcd", "length": 5, "packed": true},
```

Comment:
Decimal numbers don't need to be in quotes.  It's required for hex because the JSON format doesn't allow it.